test/language/expressions/exponentiation/exp-operator-precedence-unary-expression-semantics.js

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2015 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Rick Waldron
+email: waldron.rick@gmail.com
```

Comment:
I'm afraid the email tag [is not supported on our frontmatter](https://github.com/tc39/test262/blob/master/CONTRIBUTING.md#frontmatter).